feat(structure): create an example Unity project (#2068)
ExtendRealityLtd/VRTK: README.md

VRTK aims to make building spatial computing solutions in the [Unity] software fast and easy for beginners as well as experienced developers.

This repository contains an example Unity project showcasing how to use the [VRTK.Prefabs] package to create spatial experiences.

## Getting Started

### Downloading the project

> * Download this project repository to your local machine using one of the following methods:

Reviewer: ```suggestion
* Download this project repository to your local machine using *one* of the following methods:
```

  * Git clone the repository with `git clone https://github.com/ExtendRealityLtd/VRTK.git`

>   * or simply download the zip file at `https://github.com/ExtendRealityLtd/VRTK/archive/master.zip` and extract to your local machine.

Reviewer: ```suggestion
  * Download the zip file at `https://github.com/ExtendRealityLtd/VRTK/archive/master.zip` and extract it.
```
